lib/pbench/server/api/auth.py

```diff
@@ -79,6 +79,27 @@ def get_secret_key(self):
         except Exception as e:
             Auth.logger.exception(f"{__name__}: ERROR: {e.__traceback__}")
 
+    def verify_user(self, target_username: str) -> User:
+        """
+        Check whether the requested target user is the owner of the authorization
+        token provided to the API.
```

Comment:
I really don't think that that is what this function does -- the `User` returned by this function might have nothing to do with the authorization token if the the token grants Admin access.

What I think this function does is return the `User` associated with the target username, iff the requesting user is authorized to access it.

Reply:
That's a good summary, and actually suggests a change in the method name would probably be good. More like `get_target_user` or something?